lib/req_athena.ex

```diff
@@ -304,6 +311,29 @@ defmodule ReqAthena do
     end
   end
 
+  @credential_keys ~w(access_key_id secret_access_key region token)a
+
+  defp maybe_put_aws_credentials(request) do
+    credentials = get_credentials(request.options)
+    options = Map.take(credentials, @credential_keys)
```

Comment:
I think we don't need this line because `get_credentials` already does the `Map.take` on both branches?